src/app/app.module.ts

```diff
@@ -8,8 +8,6 @@ import { BasemapComponent } from "./basemap/basemap.component";
 
 import { MatIconModule, MatTooltipModule} from '@angular/material';
 
-import { CityioService } from "./basemap/moduleDataToMapHandler/grid/grid.service";
-import { ModuleDataToMapHandler } from "./basemap/moduleDataToMapHandler/module-data-to-map-handler.service";
```

Comment:
watching some tutorials, I see that it is no longer needed to include it in the app.module level, since services can be "lazy" loaded on demand.